### ISSUE TITLE
Declaration bundling fails on Windows

### ISSUE THREAD

**Issue by cocacola89** [wessberg/rollup-plugin-ts#19]
A dead simple project with single `index.ts` file:
```ts
export function foo() : void {  }
```
produces correct output `index.d.ts` file on Mac:
```ts
export declare function foo(): void;
//# sourceMappingURL=index.d.ts.map
```
and empty output `index.d.ts` file on Windows 10:
```ts

//# sourceMappingURL=index.d.ts.map
```

`package.json`, `tsconfig.json`, `rollup.config.js` are literally the same on both platforms.

Tried to `console.log(data)` in `program.emit(...)` callback in `flattenDeclarationsFromRollupChunk(...)` - got different results there: on Mac it does emit `export declare function foo(): void;` line, on Windows it only emits comment `//# sourceMappingURL=index.d.ts.map`.

Any ideas on what's happening and how to fix it?

**Comment by wessberg** [comment]
Hi there, thanks for reporting this! Hmm, that's very strange. Might be related to differences in filename handling. Unfortunately, I don't have a Windows machine around to test on and don't currently have a Windows VM I can spin up on my dev machine. So, it'll take some time before I can look into this issue more closely. Please, if you can tell more more, or if you are willing to look into this and open up a PR, feel free to do so.

**Comment by wessberg** [comment]
I'm investigating this issue in https://github.com/wessberg/rollup-plugin-ts/issues/21. See [this comment](https://github.com/wessberg/rollup-plugin-ts/issues/21#issuecomment-498862540) for more details.

**Comment by wessberg** [comment]
The issue has been fixed and released in v1.1.55. Thanks for reporting it!
